pyqtgraph/functions.py

```diff
@@ -1257,7 +1257,7 @@ def rescaleData(data, scale, offset, dtype=None, clip=None):
         # Cupy does not support nditer
         # https://github.com/cupy/cupy/issues/5021
 
-        data_out = data.astype(work_dtype, copy=True)
+        data_out = data.astype(work_dtype)
```

Comment:
I think `copy=True` is necessary as we are going to mutate the array in the next 2 statements

Reply:
Oh I see, `copy=True` is the default

Reply:
Oh, I remember why it was there. To make it symmetric to the `return data_out.astype(out_dtype, copy=False)`

Reply:
oh no..... I deleted the wrong `copy=True`